build.gradle

```diff
@@ -792,6 +826,16 @@ distZip {
   }
 }
 
+task distRpm {
+  dependsOn product
+  dependsOn buildRpm
+}
+
+task distDeb {
+  dependsOn product
+  dependsOn buildDeb
+}
+
```

Comment:
add ":packageVSD" dependency like being done for distTar/distZip so that VSD gets packaged even if these tasks are run independently